### DISCUSSION TITLE
Connecting after brownout

### DISCUSSION THREAD

**Discussion by PieCommando** [Chia-Network/chia-blockchain#8852] [Support]
Can anyone help?

My log says:
Cannot connect to host 120.0.0.1:8400 ssl:<ssl.SSLContext object at 0x…..[The remote computer refused connection].

GUI cannot connect. 

![9D9B639D-FA97-4465-97C5-71F97EA409A5](https://user-images.githubusercontent.com/92677930/137629999-e4504d5d-f042-4b12-baef-0dc03ec1144c.png)
![FEF59BF4-AA3D-404D-99CE-1F1AD662C20C](https://user-images.githubusercontent.com/92677930/137630002-72b433eb-0077-43d9-990b-990872ae1b1f.jpeg)

**Comment by Taweesakxch** [comment]
ส่งจาก จดหมาย <https://go.microsoft.com/fwlink/?LinkId=550986> สำหรับ
Windows



*จาก: *PieCommando ***@***.***>
*ส่งแล้ว: *17 ตุลาคม 2564 20:48
*ถึง: *Chia-Network/chia-blockchain ***@***.***>
*สําเนาถึง: *Subscribed ***@***.***>
*ชื่อเรื่อง: *[Chia-Network/chia-blockchain] Connecting after brownout
(Discussion #8852)



Caa a n snyone help?

My log says:
Cannot connect to host 120.0.0.1:8400 ssl:<ssl.SSLContext object at
0x…..[The remote computer refused connection].

GUI cannot connect.

[image: 9D9B639D-FA97-4465-97C5-71F97EA409A5]
<https://user-images.githubusercontent.com/92677930/137629999-e4504d5d-f042-4b12-baef-0dc03ec1144c.png>
[image: FEF59BF4-AA3D-404D-99CE-1F1AD662C20C]
<https://user-images.githubusercontent.com/92677930/137630002-72b433eb-0077-43d9-990b-990872ae1b1f.jpeg>

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/Chia-Network/chia-blockchain/discussions/8852>, or
unsubscribe
<https://github.com/notifications/unsubscribe-auth/AVOCTJ7UGCYREKW5IVT7O3TUHLHZHANCNFSM5GE25RMA>
.
Triage notifications on the go with GitHub Mobile for iOS
<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
or Android
<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by loppefaaret** [comment]
this might be a corrupted database - but without you `debug.log` file, and errors from it, im unable to say for sure.
you can try and move the database out, and see if that helps:
WINDOWS BLOCKCHAIN DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
mv ~\.chia\mainnet\db\block* ~\Desktop\
```
Then try opening up the Chia app again
If it worked, you can remove the files we put on the Desktop!

or you can attach you `debug.log` file, located under your users folder `~\.chia\mainnet\log\` and i can perhaps help you better resolve your issue
